docs/guides/signature-verification.md

```diff
@@ -0,0 +1,129 @@
+# Dasharo release signature verification
+
+Each published Dasharo release s signed with a signing key corresponding to
+given platform and versions. The key infrastructure is stored in
+[3mdeb-secpack](https://github.com/3mdeb/3mdeb-secpack).
+
+To verify the integrity of the binaries published in release notes on this
+site, please follow the instructions below:
+
+1. Import necessary keys to your keyring:
+
+    ```bash
+    wget https://raw.githubusercontent.com/3mdeb/3mdeb-secpack/master/keys/master-key/3mdeb-master-key.asc
+    gpg --import 3mdeb-master-key.asc
+    wget https://raw.githubusercontent.com/3mdeb/3mdeb-secpack/master/dasharo/3mdeb-dasharo-master-key.asc
+    gpg --import 3mdeb-dasharo-master-key.asc
+    wget <release signing key URL>
+    gpg --import <release signing key file>
+    gpg --list-sigs "3mdeb Master Key" "3mdeb Dasharo Master Key" <release signing key name>
+    ```
```

Comment:
I would separate commands for improved copy stability and in case `release signing key URL/file` I would mostly fill that with expected linki leaving just a minimal amount of information users have to obtain.

Reply:
And how am I supposed to do it if we have tens of keys? It is already minimal...

Reply:
Reduced it to the URL only.

Reply:
please split above in three commands

Reply:
@pietrushnic 3499a51c4842c4dddad45a977b376523aa26af12